client/src/utils/types.ts

```diff
@@ -131,10 +131,81 @@ export type TTapisApp = {
   updated: string;
 };
 
-export type TTasAllocations = {
+export type TPortalApp = {
+  loading: boolean;
+  error: {};
+  definition: {
+    id: string;
+    label: string;
+    description: string;
+    defaultQueue: string;
+    nodeCount: number;
+    coresPerNode: number;
+    maxMinutes: number;
+    tags: string[];
+  };
+  systemNeedsKeys: boolean;
+  pushKeysSystem: {};
+  execSystems: {
```

Comment:
`definition` and `execSystems` types here can be drawn from existing `TTapisApp` and `TTapisSystem` types